### DISCUSSION TITLE
support `--reuse-values`

### DISCUSSION THREAD

**Discussion by junnplus** [helmfile/helmfile#408] [Questions] [answered]
https://github.com/helmfile/helmfile/issues/221

`--reuse-values` is useful when some values are not managed by helmfile.

**Comment by yxxhero** [comment, marked as answer]
@junnplus ok. I will finish this work if you are interested in completing PR please tell me.

**Reply by yxxhero** [reply]
done  see: https://github.com/helmfile/helmfile/pull/411
